packages/compiler/CHANGELOG.json

```diff
@@ -187,7 +187,7 @@
             "comment": "Doc: Add Doc comments to built-in types and decorators"
           },
           {
-            "comment": "**Deprecation** Split `emitters` in cadl-project.yaml and compiler in 2 option `emit` and `options` that makes it consistent with the CLI"
+            "comment": "**Deprecation** Split `emitters` in tspconfig.yaml and compiler in 2 option `emit` and `options` that makes it consistent with the CLI"
```

Comment:
Do we need to keep old changelogs with names that were right at the time?

Reply:
Sounds like a good idea. In upcoming PR.